save_translation_file/models/save_translation_file.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# © 2016-TODAY Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import os
+
+from odoo import models, api, tools
+from odoo.modules import get_module_path
+from odoo.tools.misc import get_iso_codes
+
+
+class IrModuleModule(models.Model):
+    _inherit = 'ir.module.module'
+
+    @api.one
```

Comment:
`@api.one` is deprecated, use `@api.multi` with `for rec in self` loop or `self.ensure_one()`.